packages/server/src/app/app.ts

```diff
@@ -9,6 +10,17 @@ import { channelRoutes } from './routes/v2/channels';
 
 const app = new Koa();
 
+app.use(async (ctx, next) => {
+  await next();
```

Comment:
Can this be extracted to a function in its own file? It would help keep `app.ts` minimal.

Reply:
Good point. This was resolved in a later commit